src/main.cpp

```diff
@@ -3899,6 +3910,7 @@ bool static LoadBlockIndexDB()
         //   not its children.
         // - This will miss chain tips; we handle the best tip below, and other
         //   tips will be handled by ConnectTip during a re-org.
+        // TODO: sapling
```

Comment:
Remove this TODO (won't be necessary).

Reply:
What is this for?

Reply:
This loaded into memory the anchor state *after* applying a block, since we only cached in the index the state *before* applying a block, and calculating it on-the-fly would require loading the block from disk.